### PR TITLE
Initial addition of a security section.

### PR DESCRIPTION
**Pull request by kasenvr**
This can be previewed here: https://preview-docs.vircadia.dev/index.html

Initially this PR will add a data privacy section about the crash reporting in Interface.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -22,6 +22,7 @@ You have the power to shape your VR experience in Vircadia.
 * :doc:`HOST <host>` and make immersive experiences to educate, entertain, and connect with your audience.
 * :doc:`CONTRIBUTE <contribute>` to our endeavor by browsing our source code in the repository.
 * :doc:`DEVELOP <developer>` the project and tailor it to your needs, or just to help out.
+* :doc:`SECURITY <security>` information about the project and its interface.
```
Comment (ctrlaltdavid) [inline docs/source/index.rst:25]:
```suggestion
* :doc:`SECURITY <security>` information about the project and its components.```

File: docs/source/security.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+########
+Security
+########
+
+Security information about the project and its components: interface, server, and the metaverse server.
```
Comment (ctrlaltdavid) [inline docs/source/security.rst:5]:
```suggestion
Security information about the project and its components: Interface, server, and the metaverse server.
```

File: docs/source/security/reporting/crash-reporting.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Crash Reporting Privacy
+
+Your interface has crash reporting enabled by default. You can disable this in the menu **Developer > Network > Disable Crash Logger**. When this is enabled, interface will send a minidump of information to the reporting server that the Vircadia developers have access to. The current crash management server is called [Sentry](https://sentry.io/).
```
Comment (ctrlaltdavid) [inline docs/source/security/reporting/crash-reporting.md:3]:
```suggestion
Your Interface has crash reporting enabled by default. You can disable this in the menu **Developer > Network > Disable Crash Logger**. When this is enabled, Interface will send a minidump of information to the reporting server that the Vircadia developers have access to. The current crash management server is called [Sentry](https://sentry.io/).
```

File: docs/source/security/reporting/crash-reporting.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+Here is [information](https://docs.sentry.io/platforms/native/minidump/#what-is-a-minidump) on what a minidump is and how it's handled by Sentry.
+
+Here is a list of extra metadata that is collected with a crash report:
+
+* Username
+* Domain + Location
```
Comment (ctrlaltdavid) [inline docs/source/security/reporting/crash-reporting.md:10]:
```suggestion
* Domain + location
```

File: docs/source/security/reporting/crash-reporting.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+* Avatar URL
+* Build Type
+* Display Plugin
+* GPU Model + GPU Memory
+* Desktop/VR Mode
+* Machine Fingerprint
+* Metaverse Session UUID
```
Comment (ctrlaltdavid) [inline docs/source/security/reporting/crash-reporting.md:17]:
```suggestion
* Build type
* Display plugin
* GPU model + GPU memory
* Desktop/VR mode
* Machine fingerprint
* Metaverse session UUID
```

File: docs/source/security/reporting/crash-reporting.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+* "Is This A Steam Deployed Interface" Indicator
```
Comment (ctrlaltdavid) [inline docs/source/security/reporting/crash-reporting.md:18]: There's no such thing as a Steam deployed Interface now?

Comment (kasenvr) [reply docs/source/security/reporting/crash-reporting.md]: There is not, was just documenting that it gets sent. lol

**Comment by kasenvr** [comment]
> 
> 
> There seems to be an unnecessary level in the TOC tree to get to the crash reporting details page: Security > Crash Reporting > Crash Reporting Privacy
> Only the "Crash Reporting Privacy" page has content. Perhaps this content can be moved into the "Crash Reporting" page?

The reason I did that is because I was copying earlier formats since I could not get Sphinx to work with .md as natively as it does with .rst, it would not recognize the files and headers like I needed it to, so I gave up and just put an .rst middleware so that it would just work. There may be a way but I haven't found it yet.

**Comment by ctrlaltdavid** [comment]
You could copy the content of the crash-reporting.md into crash-reporting.rst and format as RST (headings, links, ...). A modified RST file: <crash-reporting.zip>

[crash-reporting.zip](https://github.com/kasenvr/vircadia-docs-sphinx/files/4768904/crash-reporting.zip)

Perhaps in the future, this page could be titled "Privacy" and cover the two privacy checkboxes in Settings > General > Privacy.

**Comment by kasenvr** [comment]
Sounds good to me, let's follow up with a PR later. :)

**Pull request merged**